app/webpack/components/authentication/sign_in.vue

```diff
@@ -296,8 +326,19 @@ export default {
       }
       return null;
     },
+    afterFailedSignIn() {
+      if (this.ui) {
+        this.ui.start(this.$refs.firebaseAuthContainer, this.uiConfig);
+      }
+      this.handleFirebaseError({ code: 'server/signin' });
```

Comment:
Given that it's won't necessarily just be a Firebase error, suggest renaming `handleFirebaseError` --> `handleError` (since it is pretty much a catch all error handler for any known error).

Reply:
Done!